modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -8,6 +8,67 @@ For performing backups, Neo4j uses the _Admin Service_, which is only available
 For more information, see xref:kubernetes/accessing-neo4j.adoc[Accessing Neo4j].
 ====
 
+[[kubernetes-backup-modes]]
+== Backup modes
+
+Neo4j's Helm chart supports two backup modes:
+
+=== Cloud provider mode
+
+Cloud provider mode uses Neo4j's native cloud storage integration with direct upload to create immutable backup objects.
+
+* *Supported providers*: AWS S3, Google Cloud Storage, and Azure Blob Storage.
+* *Benefits*: No persistent volume requirements. Supports differential backups and immutable backup objects.
+* *Configuration*: Set `cloudProvider` to `aws`, `gcp`, or `azure`.
+
+=== Local mode
+
+Local mode creates local backups in `/backups` mount.
+
+* *Requirements*: Persistent storage for large databases (configured via `tempVolume`).
+* *Configuration*: Leave `cloudProvider` empty.
+
+[[kubernetes-cloud-native-features]]
+== Cloud-native backup features
+
+When using cloud providers, Neo4j's native backup provides:
+
+* *Direct cloud storage upload* - No intermediate local storage required.
+* *Differential backup chains* with `preferDiffAsParent: true`.
+* *Immutable backup objects* in cloud storage.
+* *Support for S3-compatible endpoints*.
+* *Enhanced S3 configuration* including custom CA certificates and endpoint settings.
+
+[[kubernetes-differential-backups]]
+== Differential backups
```

Comment:
Do we need this section at all? In the examples below, `preferDiffAsParent: true` and `type: "AUTO"`. I think we can just add a note to remind them about this, but they can look at the section Configure the backup parameters and see all required parameters, not just a snippet of them.

Reply:
I would suggest that we add the following note on line 60: 
```
[NOTE]
====
When using cloud providers, differential backups do not require persistent volumes with previous backups.
Instead, the chart will first create a full backup in the cloud storage, and then, the subsequent backups will be differential backups that reference this full backup.
====

```